src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -128,7 +130,8 @@ public User getUser(String email) {
             return null;
         }
         String aboutMe = (String) userEntity.getProperty("aboutMe");
-        User user = new User(email, aboutMe);
+        long level = (long) userEntity.getProperty("level");
+        User user = new User(email, aboutMe, (int) level);
```

Comment:
Could you explain to me the change of level between int and long?

Reply:
I am not sure the reasoning behind it but i know that it breaks a few things if i just convert it to an int. (that is a good question though and worth looking into more but i dont have a solid explanation) - What happens if i just save it directly as an int is i get a 'java.lang.ClassCastException: java.lang.Long cannot be cast to java.lang.Integer' error but the way i did it above is fine for some reason... even though level is never saved as a long type.